assopy/auth_backends.py

```diff
@@ -57,8 +57,9 @@ def authenticate(self, uid=None):
             return None
 
 class EmailBackend(_AssopyBackend):
-    def authenticate(self, email=None, password=None):
+    def authenticate(self, request, email=None, password=None, username=None):
         try:
```

Comment:
funny how it already checked for `email` anyway. ;)

Reply:
That's because the `assopy.forms.LoginForm` used for normal user authentication uses an `email` field - so it works correctly there; but admin login form passes a `username`, so the `email` field was blank.